tensorflow/core/platform/default/logging.cc

```diff
@@ -534,9 +553,11 @@ void TFDefaultLogSink::Send(const TFLogEntry& entry) {
   }
 
   // TODO(jeff,sanjay): Replace this with something that logs through the env.
-  fprintf(stderr, "%s.%06d: %c%s %s:%d] %s\n", time_buffer, micros_remainder,
-          sev, tid_buffer, entry.FName().c_str(), entry.Line(),
-          entry.ToString().c_str());
+  mutex mu;
+  mutex_lock l(mu);
```

Comment:
This lock isn't needed. By the time this point is reached, the `VlogFileMgr` will be initialized, and fprintf is thread-safe.